### PR TITLE
Pr #3

### PR DESCRIPTION
**Pull request by jamezmca**
Using Cloudinary to host images where the fetched responses from Cloudinary are passed into postMessage from the Input.js file.  After being passed into postMessage, the images are passed through the Messages.js file into the SenderBubble.js file. The styling is dependent on the number of images sent and whether or not text is included in the message :P

Screen Shot 2021-09-11 at 5 46 23 PM
Screen Shot 2021-09-11 at 5 48 43 PM

Screen Shot 2021-09-11 at 5 50 28 PM

### CODE REVIEW

**Review by sonipranjal — COMMENTED** [review]
Good job!

- Always remember to clean up the code before making a pull request.
- Now, we can select and upload one image at a time, but as described in mocks, we should be able to select more than one image.

I have added some comments below in more detail; please address them and let us know on the slack channel when this PR is ready for another review!

**Review by etchao — APPROVED** [review]
There are still a couple areas that could be improved and the UI for sending images is not consistent with the spec (the spec shows the text below the image if both are present), but otherwise this PR can be merged 🙂

File: client/src/components/ActiveChat/Input.js [hunk 1]
```diff
@@ -1,26 +1,69 @@
...
 import { makeStyles } from "@material-ui/core/styles";
 import { connect } from "react-redux";
 import { postMessage } from "../../store/utils/thunkCreators";
+import FileCopyIcon from '@material-ui/icons/FileCopy';
 
 const useStyles = makeStyles(() => ({
   root: {
     justifySelf: "flex-end",
-    marginTop: 15
+    marginTop: 15,
```
[the remark this answers is not in the document]
Comment (jamezmca) [reply client/src/components/ActiveChat/Input.js:11]: Thanks for that comment - I was unaware but I will be sure to implement this in future. Also regarding the text with image sending, I did implement the text displaying beneath the image for the case where only a single image is sent. If two images are sent, it displays the text above. Does it work the same for you?

Thanks again for your feedback :)

[the remark this answers is not in the document]
Comment (etchao) [reply client/src/components/ActiveChat/Input.js:11]: Ah yes - my mistake about the UI, it looks right to me. I made the mistake of testing with a white image and saw it incorrectly. Sorry about that!

[the remark this answers is not in the document]
Comment (jamezmca) [reply client/src/components/ActiveChat/Input.js:11]: No worries at all :) thanks for your feedback!

**Pull request merged**